### PR TITLE
[Package Add] freetube-deb

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

**Comment by wizard-28** [comment]
Shouldn't it be `freetube-deb`?

**Review by Henryws — CHANGES_REQUESTED** [review]
yeah, what @wizard-28 said

**Review by wizard-28 — CHANGES_REQUESTED** [review]
I think it also needs to have a `gives` field, for it to work.

See [this comment](https://github.com/pacstall/pacstall-programs/pull/69/files#r671796341) by @NarutoXY.

And @NarutoXY it will be helpful if you add the `deb` pacscript making instructions to [`make-a-pacscript.md`](https://github.com/pacstall/pacstall-programs/blob/master/make-a-pacscript.md)

**Comment by Henryws** [comment]
> I think it also needs to have a `gives` field, for it to work.
> 
> See [this comment](https://github.com/pacstall/pacstall-programs/pull/69/files#r671796341) by @NarutoXY.
> 
> And @NarutoXY it will be helpful if you add the `deb` pacscript making instructions to [`make-a-pacscript.md`](https://github.com/pacstall/pacstall-programs/blob/master/make-a-pacscript.md)

Why would we need a `gives` field? Would we hook into apt somehow?

**Comment by wizard-28** [comment]
> Why would we need a gives field? Would we hook into apt somehow?

See @NarutoXY's comment, I think he is the one who added `.deb` support so, he should know the best.

**Comment by NarutoXY** [comment]
Hey we are currently using give field for removing package as in my mind it's always the name by which apt installs any deb package and to remove it we need that name. We can use apt list then search then what not but it'll be complicated. Never work and it will also be culprit of deleting wrong program.

File: packages/freetube-deb/freetube-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name="freetube-deb"
+version="0.13.2"
+url="https://github.com/FreeTubeApp/FreeTube/releases/download/v${version}-beta/freetube_${version}_amd64.deb"
```
Comment (NarutoXY) [inline packages/freetube-deb/freetube-deb.pacscript:3]: Add ```gives=freetube```

**Review by NarutoXY — APPROVED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**